Actions/Copy.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using UnityEditor;
-using UnityEngine.Experimental.EditorVR.Utilities;
 
 namespace UnityEngine.Experimental.EditorVR.Actions
 {
 	[ActionMenuItem("Copy", ActionMenuItemAttribute.kDefaultActionSectionName, 5)]
+	[Tooltip("Copy")]
```

Comment:
Since actions seem to be the only one that make use of this attribute, I think we can get rid of the attribute and fold the tooltip into the ActionMenuItem attribute as an additional parameter and/or simply use the text.